website/content/community/code-of-conduct.md

```diff
@@ -58,11 +58,12 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at dave@natulte.net. All
-complaints will be reviewed and investigated and will result in a response that
-is deemed necessary and appropriate to the circumstances. The project team is
-obligated to maintain confidentiality with regard to the reporter of an incident.
-Further details of specific enforcement policies may be posted separately.
+reported by contacting Rodrigo Campos at rodrigocc@gmail.com and Dax McDonald
+at daxmc99@gmail.com (please mail both addresses). All complaints will be
+reviewed and investigated and will result in a response that is deemed
+necessary and appropriate to the circumstances. The project team is obligated
+to maintain confidentiality with regard to the reporter of an incident. Further
+details of specific enforcement policies may be posted separately.
```

Comment:
>Further details of specific enforcement policies may be posted separately.

I'm not sure what the purpose of this sentence is. Shouldn't we just add more details if/when we have them?

Reply:
That's from the text of the contributor covenant, the template used here.  See the links at the bottom of the doc.

I think it's just leaving it open for the project to define some additional policies for how enforcement may be handled.  I don't think we need to do anything though.

Reply:
@russellb +1